packages/firecamp-echo-server/src/rest/rest.controller.ts

```diff
@@ -1,5 +1,41 @@
 import * as rawBody from 'raw-body';
-import { BadRequestException, Body, Controller, Get, Headers, Param, Post, Put, Query, Req, Response, HttpStatus, HttpCode, Patch, Delete } from '@nestjs/common';
+import { BadRequestException, Body, Controller, Get, Headers, Param, Post, Put, Query, Req, Response, HttpStatus, HttpCode, Patch, Delete, Head, Ip, Header, StreamableFile } from '@nestjs/common';
+import * as crypto from 'crypto'
+import { response } from 'express';
+import * as Hawk from 'hawk'
+import { Credentials } from 'hawk/lib/server';
+import { createReadStream } from 'fs';
+import { join } from 'path';
+import {createDeflate, createGzip} from 'zlib';
+import { Readable } from 'stream';
+
+const echo_username = 'firecamp'
+const echo_password = 'password'
+const hawk_id = 'dh37fgj492je'
```

Comment:
You're using two different patterns -- above here you don't have a trailing `;`, below here you do.

If you did something like:

```suggestion
const hawk_id = 'dh37fgj492je' /* check-spelling: token-ok */;
```

You could then add:
```
# allow a single quoted token
'[^']+'\s+/\* check-spelling: token-ok \*/
```
to the `.github/actions/spelling/patterns.txt` file

Note that I have no particular preference for how such a pattern is written, this would work.

So would regularizing the token before the `=` so that you could do something like:

```
# accept constant token values
const (?:hawk_id|consumer\w+) = '[^']+'
```

-- Note that this example won't work for the `key:` item below, but you could structure something for it if you prefer this pattern.